01 - JavaScript Drum Kit/index-START.html

```diff
@@ -1,66 +1,122 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>JS Drum Kit</title>
-  <link rel="stylesheet" href="style.css">
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <title>JS Drum Kit</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="keys">
```

Comment:
I always love screenshots in a pull request. maybe this may be helpful here. (if you do this in your first pull request in the new company, your team mates will be impressed :))

Reply:
What do you mean exactly with a screenshot here? I would like to understand it better to impress my new teammates!!

Reply:
Maybe we can talk about it on Thursday?

Reply:
sure we can talk about it on thursday. what I meant is just a browser screenshot of the website you made.